modern/src/EditItemView.js

```diff
@@ -56,23 +56,32 @@ const EditItemView = ({ children, endpoint, item, setItem }) => {
   };
 
   return (
-    <>
-      <MainToolbar />
-      <Container maxWidth='xs' className={classes.container}>
+    <OptionsLayout>
+      <Container maxWidth="xs" className={classes.container}>
         {children}
-        <FormControl fullWidth margin='normal'>
+        <FormControl fullWidth margin="normal">
           <div className={classes.buttons}>
-            <Button type='button' color='primary' variant='outlined' onClick={() => history.goBack()}>
+            <Button
+              type="button"
+              color="primary"
+              variant="outlined"
+              onClick={() => history.goBack()}
+            >
```

Comment:
Another styling nit. We keep `>` on the last line. I'm happy to change, but need to update all other places.